src/sql/workbench/services/notebook/browser/outputs/factories.ts

```diff
@@ -55,12 +55,26 @@ export const textRendererFactory: IRenderMime.IRendererFactory = {
 	mimeTypes: [
 		'text/plain',
 		'application/vnd.jupyter.stdout',
-		'application/vnd.jupyter.stderr'
+		'application/vnd.jupyter.stderr',
+		'application/vnd.code.notebook.stdout',
+		'application/vnd.code.notebook.stderr'
 	],
 	defaultRank: 120,
 	createRenderer: options => new widgets.RenderedText(options)
 };
 
+/**
+ * A mime renderer factory for VS Code Notebook error data.
```

Comment:
The error mime type is returned as a json object, rather than just text. So if we use the same text widget then it outputs as raw json. This error widget adds some additional formatting for better readability. VS Code already does something similar with error types in their rendering code.